apps/cmp.c

```diff
@@ -1210,7 +1162,7 @@ static STACK_OF(X509) *load_certs_multifile(char *files,
 
         if (!load_certs_autofmt(files, &certs, 0, pass, desc))
             goto err;
-        if (!sk_X509_add1_certs(result, certs, 0, 1 /* no dups */, 0))
+        if (!OSSL_sk_X509_add_certs(result, certs, 1, 0, 1 /* no dups */, 0))
```

Comment:
I have to wonder whether this is the right name. In particular the prefix `OSSL_sk` looks kind of odd. The `sk_` prefix has generally been reserved for the low level stack manipulation functions. This function isn't really at that level. I looked to see what we've named other higher level things that take a stack as the main parameter. Here are some examples:

````
X509 *X509_find_by_subject(STACK_OF(X509) *sk, const X509_NAME *name);
STACK_OF(X509) *X509_chain_up_ref(STACK_OF(X509) *chain);
STACK_OF(X509_ATTRIBUTE) *X509at_add1_attr(STACK_OF(X509_ATTRIBUTE) **x,
                                           X509_ATTRIBUTE *attr);
````

So, I'm wondering whether simply `X509_add_certs` might be good enough? Similarly with the other function introduced by this PR.

Reply:
I'm happy to remove the `OSSL_sk` prefix from the function names; done.